.ci/pipeline/ci.yml

```diff
@@ -33,13 +37,13 @@ jobs:
     release.dir: '__release_lnx_gnu'
     platform.type : 'lnx32e'
   pool:
-    vmImage: 'ubuntu-20.04'
+    vmImage: 'ubuntu-22.04'
   steps:
-  - script: sudo apt-get update && sudo apt-get install gcc-multilib g++-multilib openjdk-8-jdk
+  - script: sudo apt-get update && sudo apt-get install gcc-multilib g++-multilib
     displayName: 'apt-get'
   - script: |
-      .ci/scripts/build.sh --platform $(platform.type) --target daal
-    displayName: 'make daal'
+      .ci/scripts/build.sh --platform $(platform.type) --target daal_c
+    displayName: 'make daal_c'
```

Comment:
Could you please provide some clarification why we are removing java interface validation?